lib/util/get-typescript-extension-map.js

```diff
@@ -117,10 +117,12 @@ module.exports = function getTypescriptExtensionMap(context) {
         get(context.settings?.n ?? context.settings?.node) ||
         getFromTSConfigFromFile(
             // eslint ^8
-            context.physicalFilename ??
+            context.getPhysicalFilename() ??
+                context.physicalFilename ??
                 // eslint ^7.28 (deprecated ^8)
                 context.getPhysicalFilename?.() ??
                 // eslint ^8 (if physicalFilename undefined)
+                context.getFilename() ??
                 context.filename ??
                 // eslint ^7 (deprecated ^8)
                 context.getFilename?.()
```

Comment:
**question:** Where these ones not already working?

Also, can you fix the comments if this is the corerct way around

Reply:
```suggestion
            context.physicalFilename ??
                // eslint ^7.28 (deprecated ^8)
                context.getPhysicalFilename?.() ??
                // eslint ^8 (if physicalFilename undefined)
                context.filename ??
                // eslint ^7 (deprecated ^8)
                context.getFilename?.()
```